lib/checkPullRequestLabels.js

```diff
@@ -363,3 +365,34 @@ module.exports.checkForIssueLabel = async function (context) {
     await context.github.issues.removeLabel(removeLabelParams);
   }
 };
+
+/**
+ * This function checks and prevent the removal of stale build label
+ * if branch is not updated.
+ *
+ * @param {import('probot').Context} context
+ */
+module.exports.checkStaleBuildLabelRemoved = async function (context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+  const pullRequest = context.payload.pull_request;
+  if (label.name === OLD_BUILD_LABEL) {
+    const user = context.payload.sender;
+
+    if (checkPrIsNotStale(pullRequest, context)) {
```

Comment:
We are checking whether the PR is stale right?
```suggestion
    if (checkPrIsStale(pullRequest, context)) {
```

Reply:
addressed in latest commit.